ramalama/common.py

```diff
@@ -282,9 +282,19 @@ def get_gpu():
         os.environ["HIP_VISIBLE_DEVICES"] = str(gpu_num)
         return
 
+    # INTEL iGPU CASE (Look for ARC GPU)
+    if os.path.exists('/sys/bus/pci/drivers/i915/0000:00:02.0/device'):
+        try:
+            with open('/sys/bus/pci/drivers/i915/0000:00:02.0/device', 'rb') as f:
```

Comment:
Would it make sense to use a glob for this?

glob.glob(/sys/bus/pci/drivers/i915/*/device)

Reply:
I'll try that.

Learning Python on the Go as it were...  ;-)

Reply:
Modified to use glob.

WDYT?